src/test/java/org/mule/tools/apikit/MainAppScaffolderWithExistingConfigApiSyncTest.java

```diff
@@ -112,21 +112,21 @@ public void reScaffoldApiSyncWithGlobalWithoutChanges() throws Exception {
 
 
   @Test
-  public void reScaffoldLossesDisableValidations() throws Exception {
```

Comment:
The changes look good.

Was there a **good** reason for having this test before? Given that we have an explicit test saying _"these things will be lost"_ it sounds fishy to change behavior without understanding why it was there.

Reply:
Oh wait i'm dumb. The test didn't asserted the property was properly preserved 🤦.

bad naming can certainly bite you